src/store/modules/product/getters.ts

```diff
@@ -5,6 +5,9 @@ import RootState from '../../RootState'
 const getters: GetterTree <ProductState, RootState> = {
   getCurrent (state) {
     return state.current
+  },
+  getProduct: (state) => (productId: string) => {
+    return state.current[productId] ? state.current[productId] : 0
   }
 }
-export default getters
\ No newline at end of file
+export default getters;
```

Comment:
Remove unwanted change

Reply:
Removed unwanted changes.
Thank you sir for reviewing my code.